pkg/testcase/networkconnectivity.go

```diff
@@ -31,6 +31,19 @@ func TestInternodeConnectivityMixedOS(deleteWorkload bool) {
 	}
 }
 
+// TestCIDRDualStack Validates Pod IPs and Cluster IPs in CIDR range
```

Comment:
the doc name should be the same as the func

Reply:
Updated